src/lib/streamSpecToRegistry.ts

```diff
@@ -24,7 +24,14 @@ export default async function streamSpecToRegistry(spec: string) {
 
   debug('file and stream created, streaming into form data payload');
   const formData = new FormData();
-  formData.append('spec', stream);
+  formData.append('spec', {
+    type: 'application/json',
+    name: 'openapi.json',
+    [Symbol.toStringTag]: 'File',
+    stream() {
+      return stream;
+    },
+  });
```

Comment:
@erunion I'd like your blessing with these changes now that we're using `formdata-node` — does our API care what we name the file? I just set it to `openapi.json` but let me know if you feel otherwise.